src/lock.rs

```diff
@@ -18,7 +18,7 @@ pub struct Lock {
     /// The LLVM repository URL.
     pub url: String,
     /// The LLVM repository branch.
-    pub branch: String,
+    pub branch: Option<String>,
```

Comment:
Why made optional?
Please add a comment.

Reply:
Added a comment: "Optional to allow the default branch clone and checkout"

Reply:
Using the default branch is extremely dangerous as you are risking to *silently* link the binary against a wrong LLVM commit and then you can a totally bytecode.
In our release and building process, we're only using exact versions/branches.
Please revert to a mandatory arg.

Reply:
Reverted back.